test/number/sign.wren

```diff
@@ -0,0 +1,6 @@
+IO.print(123.sign)      // expect: 1
+IO.print((-123).sign)   // expect: -1
+IO.print(0.sign)        // expect: 0
+IO.print((-0).sign)     // expect: 0
```

Comment:
Yeah, I think I'd expect it to follow `signbit()`. Negative zero does feel pretty squishy to me, but IEEE doubles support it and Wren follows that so my normal guideline is to just assume the C standard lib folks have put more thought into this than I have so just follow what they do. :)